### PR TITLE
Added Bootstrap and EESSI Overlay config to README

### PR DESCRIPTION
**Pull request by peterstol**
My first pull request. Added Install documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,42 @@
...
+
+## Installation and Configuration
+
+### Prerequisites
+
+The bootstrap process will need a clean environment with a compiler (the system version of gcc will do). It also is very sensitive to 
+the environment, so setup a user with unset CFFLAGS, CFLAGS, LDFLAGS, PKG_CONFIG_PATH and the always harmful LD_LIBRARY_PATH variables.
+EESSI provides a singularity container for this.
+
+## Installation and Configuration
```
Comment (bedroge) [inline README.md:17]: There are two section named "Installation and Configuration". Probably best to merge them by moving the subsection Prerequisites to the second section?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,42 @@
 # Compatibility layer
 
+## Introduction
+
 The compatibility layer of the EESSI project uses [Gentoo Prefix](https://wiki.gentoo.org/wiki/Project:Prefix)
+to provide a known base on top of the host. This is the foundation we use to build our software stack on.
+An alternative would be the NixOS.
```
Comment (bedroge) [inline README.md:7]: Add URL to NixOS?

Comment (bedroge) [inline README.md]:
```suggestion
An alternative would be the [NixOS](https://nixos.org/).
```

File: README.md (hunk 1 above)
Comment (bedroge) [inline README.md:17]:
```suggestion
```

File: README.md (hunk 2 above)
Comment (peterstol) [reply README.md]: done

**Pull request merged**